skyportal/handlers/api/sysinfo.py

```diff
@@ -56,37 +59,25 @@ def get(self):
                     "git",
                     "--git-dir=.git",
                     "log",
-                    "--pretty=format:'%C(cyan)[%ci]%Creset %s %C(auto)%h'",
+                    "--no-merges",
+                    "--first-parent",
+                    "--pretty=format:[%ci %h] %s",
+                    f"-{default_log_lines}",
                 ],
                 capture_output=True,
                 universal_newlines=True,
             )
             loginfo = p.stdout
 
-        raw_gitlog = loginfo.splitlines()[:default_log_lines]
-        gitlog = []
+        raw_gitlog = loginfo.splitlines()
```

Comment:
Can we filter out commits from dependabot?

Reply:
Done.